pyproject.toml

```diff
@@ -22,6 +22,7 @@ classifiers = [
 keywords = ["lsst"]
 dependencies = [
     "lsst-felis @ git+https://github.com/lsst/felis.git@main",
+    "lsst-sdm-tools @ git+https://github.com/lsst/sdm_tools.git@main",
```

Comment:
Please remove the git+https here and above. You can not put non-PyPI URLs in the pyproject.toml dependencies. We have them in the requirements.txt to make it easy to install the current main in our CI builds.

Reply:
Just listing `lsst-sdm-tools` here without the GitHub URL will not work though, because it is not being deployed to PyPI. (That could be changed though.)

Reply:
Locally, `pip install .` works fine with these URLs in `pyproject`, but I do understand that this does not work with PyPI. 

sdm_schemas is not deployed to PyPI, and I believe we have no plans to do this.

Reply:
Are you saying that because you've tried it and it didn't work? I can't see why it would fail because higher up the package has already been installed. The dependencies will not be installed from pyproject.toml because you have already installed them and it knows that lsst-sdm-tools is available (and also with `pip install --no-deps`  it won't even try).

Reply:
I see what you are saying. I'll change these and remove the GitHub URLs.